src/analyze/analyze-cat-config.c

```diff
@@ -23,7 +22,7 @@ int verb_cat_config(int argc, char *argv[], void *userdata) {
                         print_separator();
 
                 if (path_is_absolute(*arg)) {
-                        NULSTR_FOREACH(dir, CONF_PATHS_NULSTR("")) {
+                        FOREACH_STRING(dir, CONF_PATHS("")) {
```

Comment:
in theory enumerating through a nulstr is supposed to be nicer, because faster as it means we only have a single dynamic reference to resolve, rather than a bunch. search through a string is something CPUs are really good at, but symbold resolution is not really what they are good at.

but also, i don't really care, this is not really perfromance sensitive.